spk/postgresql/Makefile

```diff
@@ -0,0 +1,24 @@
+SPK_NAME = postgresql
+SPK_VERS = 11.6
+SPK_REV = 1
+SPK_ICON = src/postgresql.png
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = DigitalBox98
+DESCRIPTION = PostgreSQL is a powerful, open source object-relational database system with over 30 years of active development that has earned it a strong reputation for reliability, feature robustness, and performance
+
+RELOAD_UI = yes
+DISPLAY_NAME = PostgreSQL
+STARTABLE = yes
+
+HOMEPAGE   = http://www.postgresql.com
+LICENSE    = PostgreSQL Licence
+
+WIZARDS_DIR = WIZARD_UIFILES
+
+INSTALLER_SCRIPT = src/installer.sh
```

Comment:
In my opinion you should keep `SERVICE_SETUP` variable and "service_preinst", "service_postinst" previous version of functions... Here package has lost many common framework support and it does not make sense to copy-paste them.
Please also take care to your scripts space indentation.